recipes/default.rb

```diff
@@ -162,6 +162,10 @@
  raise "The leader election protocol has a higher timeout than the transaction timeout in NDB. We can get false suspicions for a live leader. Invalid configuration."
 end
 
+if node['hops']['nn']['direct_memory_size'].to_i < node['hops']['nn']['heap_size'].to_i
+  raise "Invalid Configuration. Set Java DirectByteBuffer memory as high as Java heap size otherwise, the NNs might experience severe GC pauses."
+end
```

Comment:
This check should be in `install` recipe, fail fast

Reply:
moved to install.rb